internal/k8s/controller.go

```diff
@@ -2332,7 +2332,7 @@ func (lbc *LoadBalancerController) updateTransportServersStatusFromEvents() erro
 			events, err := lbc.client.CoreV1().Events(ts.Namespace).List(context.TODO(),
 				meta_v1.ListOptions{FieldSelector: fmt.Sprintf("involvedObject.name=%v,involvedObject.uid=%v", ts.Name, ts.UID)})
 			if err != nil {
-				allErrs = append(allErrs, fmt.Errorf("error trying to get events for TransportServer %v/%v: %w", ts.Namespace, ts.Name, err))
+				allErrs = append(allErrs, fmt.Errorf("trying to get events for TransportServer %v/%v: %w", ts.Namespace, ts.Name, err))
```

Comment:
Instead of removing the word `error` we can replace it with `unable to` or `failed to`.

Right now the sentence makes me think that the application is still trying to get events after displaying the message